packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -52,6 +56,9 @@ const MarketPlacePage = () => {
   const [npmPackageType, setNpmPackageType] = useState('plugin');
   const { autoReload: isInDevelopmentMode, dependencies, useYarn } = useAppInfos();
   const isOnline = useNavigatorOnLine();
+  const { search } = useLocation();
+  const params = useMemo(() => new URLSearchParams(search), [search]);
+  const sort = params.get('sort') || 'name:asc';
```

Comment:
The helper plugin provides an abstraction that you can use to avoid this (see an example [here](https://github.com/strapi/strapi/blob/5dfaf2a3f88a42c9bd57001b873124c93bef171e/packages/core/admin/admin/src/pages/MarketplacePage/index.js#L56))